openslides_backend/migrations/migrations/0032_fix_motion_origin_relations.py

```diff
@@ -63,4 +70,26 @@ def migrate_event(
                         )
                     )
             return new_events
+        elif isinstance(event, UpdateEvent):
+            """Changes the update event
+            Precondition from backend: The backend sorts the events by type: create/update/delete
+            Therefore we don't need the self.get_additional_events-method and
+            can modify the original event.
+            To fix the current migration deleted motions will be removed from the update event
+            """
+            if "all_derived_motion_ids" in event.data and self.created_motion_ids.get(
+                id
+            ):
+                new_derived_motion_ids: List[int] = []
+                for motion_id in event.data["all_derived_motion_ids"]:
+                    if motion_id in self.created_motion_ids[id]:
```

Comment:
Why do you need the `created_motion_ids`? Can't you just drop it altogether and just iterate over the fields and check if the motion was deleted?

Reply:
this way i could be sure that the correct newly created motion is allowed in update. The created motion is still not applied to keyframe_models.
And it is cheaper, because don't need to ask the database